src/CentrifugeRouter.sol

```diff
@@ -176,6 +202,10 @@ contract CentrifugeRouter is Auth, ICentrifugeRouter {
         return IPoolManager(poolManager).getVault(poolId, trancheId, asset);
     }
 
+    function estimate(bytes calldata payload) external returns (uint256 amount) {
```

Comment:
I don't think we need to expose that function here. Users can use the gateway directly

Reply:
We also discussed that. Since we would like to have the CentifugeRouter as a one-stop shop, we would like for the integrators / users to care only about this contract and nothing else.